addons/website_event_questions/static/src/js/website_event_questions.js

```diff
@@ -1,16 +0,0 @@
-odoo.define('website_event_questions.website_event_questions', function (require) {
-'use strict';
-
-var EventRegistrationForm = require('website_event.website_event');
-
-EventRegistrationForm.include({
```

Comment:
Dead JS ?

Reply:
Used to generate a select box with select2.
I switched to default BS select. Is it a problem?

Reply:
Nope, cool :+1: